ui/tests/helpers/general-selectors.ts

```diff
@@ -53,8 +53,8 @@ export const GENERAL = {
     input: (attr: string) => `[data-test-ttl-value="${attr}"]`,
   },
 
-  validation: (attr: string) => `[data-test-field-validation=${attr}]`,
-  validationWarning: (attr: string) => `[data-test-validation-warning=${attr}]`,
+  validation: (attr: string) => `[data-test-form-field-validation-error=${attr}]`,
```

Comment:
```suggestion
  validation: (attr: string) => `[data-test-validation-error=${attr}]`,
```

Reply:
for consistency, I've called it `validationErrorByAttr`(since the other one is `validationWarningByAttr`)